scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -311,26 +311,21 @@
   ; At this point, we don't care about the launch pod or the metric timers, so toss their dictionary away.
   {:cook-expected-state :cook-expected-state/running})
 
-(def get-pod-ip->hostname-fn
-  (memoize
-    (fn [pod-ip->hostname-fn]
-      (if pod-ip->hostname-fn (util/lazy-load-var pod-ip->hostname-fn) identity))))
-
 (defn record-sandbox-url
   "Record the sandbox file server URL in datomic."
   [pod-name {:keys [pod]}]
   (when-not (api/synthetic-pod? pod-name)
     (let [task-id (-> pod .getMetadata .getName)
           pod-ip (-> pod .getStatus .getPodIP)
-          {:keys [default-workdir pod-ip->hostname-fn sidecar]} (config/kubernetes)
+          {:keys [default-workdir sidecar]} (config/kubernetes)
           sandbox-fileserver-port (:port sidecar)
           sandbox-health-check-endpoint (:health-check-endpoint sidecar)
           sandbox-url (try
                         (when (and sandbox-fileserver-port
                                    sandbox-health-check-endpoint
                                    (not (str/blank? pod-ip)))
                           (str "http://"
-                               ((get-pod-ip->hostname-fn pod-ip->hostname-fn) pod-ip)
+                               pod-ip
```

Comment:
Is this unrelated to the rest of the PR?

Reply:
yeah, just was looking at usage of lazy load and cleaned this up. so related a tiny bit